public/pug/landing.pug

```diff
@@ -4,42 +4,30 @@ block styles
     link(rel="stylesheet" type="text/css" href="../css/landing.css")
 
 block main
-    section(class="container")
-        h1  Be more productive
-        p.
-            "What to do" is an easy to use to-do application, designed for
-            individual or colaborativ use.
+    .container
+        p#logo.landing What to do?
         .section-features
-            .features.lists
-                p.
-                    Todos can be grouped into lists in order to sort them.
-                    You want to have seperate lists for work, home and shopping
-                    or a project is around the corner? Sort your task to be able 
-                    to focus on what matters for you.
-                a( href="/welcome" class="btn" )
-                    .inline-items
-                        p learn more
-                        img(src="/assets/img/tick.svg")
-            .features.share
-                p.
-                    List and single todos can be shared with other contributors.
-                    Have a shopping list for your partner or organize the next
-                    gathering and assign task in order to never lose oversight
-                    again.
-                a( href="/welcome" class="btn" )
-                    .inline-items
-                        p learn more
-                        img(src="/assets/img/tick.svg")
-            .features.multimedia
-                p.
-                    Todos can have images, text and recordings added to them.
-                    Sometimes a picture can tell more then a thousand words.
-                    We enable you to use whatever you need to be as effective
-                    as you wish to be.
-                a( href="/welcome" class="btn" )
-                    .inline-items
-                        p learn more
-                        img(src="/assets/img/tick.svg")
-
-block scripts
-    script( scr='../vue.js')
+            .features#lists
+                .inline-items
+                    p.
+                        Group tasks into lists in order to sort them.
+                        Want to have seperate lists for work, home and shopping
+                        or a project is around the corner? Sort your tasks to be able 
+                        to focus on what matters.
+                    img.tick(src="/assets/img/tick.svg")
```

Comment:
What function do the ticks serve now?

Reply:
Both the color-coding and the ticks should compliment the text in representing the functionality of the app by not only describing the feature set but showing visually how task might be represented